CODEOWNERS

```diff
@@ -193,18 +181,83 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
+# compression code
+/*/extensions/filters/http/decompressor @rojkov @kbaichoo
+/*/extensions/filters/http/gzip @gsagula @kbaichoo
+/*/extensions/filters/http/common/compressor @gsagula @rojkov @kbaichoo
+/*/extensions/filters/http/compressor @rojkov @kbaichoo
+/*/extensions/compression/brotli @rojkov @kbaichoo
+/*/extensions/compression/common @kbaichoo @rojkov
+/*/extensions/compression/gzip/decompressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/compressor @rojkov @kbaichoo
+/*/extensions/compression/gzip/common @rojkov @kbaichoo
+# cel
+/*/extensions/access_loggers/filters/cel @kyessenov @douglas-reid @adisuissa
+# health cehck
+/*/extensions/filters/http/health_check @mattklein123 @adisuissa
+# lua
+/*/extensions/filters/http/lua @mattklein123 @wbpcode
+/*/extensions/filters/common/lua @mattklein123 @wbpcode
+# rbac
+/*/extensions/filters/network/rbac @yangminzhu @yanavlasov
+/*/extensions/filters/http/rbac @yangminzhu @yanavlasov
+/*/extensions/filters/common/rbac @yangminzhu @yanavlasov
+# tap
+/*/extensions/filters/http/tap @mattklein123 @daixiang0
+/*/extensions/common/tap @mattklein123 @daixiang0
+/*/extensions/transport_sockets/tap @mattklein123 @daixiang0
+# local rate limit
+/*/extensions/filters/network/local_ratelimit @mattklein123 @wbpcode
+
+# "Underowned" extensions (no backup reviewers, please volunter!)
+# Stateful session
+/*/extensions/http/stateful_session/cookie @wbpcode @UNOWNED
+/*/extensions/filters/http/stateful_session @wbpcode @UNOWNED
+# stats
+/*/extensions/stat_sinks/statsd @mattklein123 @UNOWNED
+/*/extensions/stat_sinks/common @mattklein123 @UNOWNED
+/*/extensions/stat_sinks/common/statsd @mattklein123 @UNOWNED
+# proxy protocol
+/*/extensions/filters/listener/proxy_protocol @ggreenway @UNOWNED
+# cors
+/*/extensions/filters/http/cors @wbpcode @UNOWNED
+# access loggers
+/*/extensions/access_loggers/grpc @wbpcode @UNOWNED
+/*/extensions/access_loggers/file @wbpcode @UNOWNED
+
+# "Unowned" extensions (currently without 1 maintainer or 2 owners)
+/*/extensions/formatter/req_without_query @UNOWNED @tsaarni
+# ext_authz
+/*/extensions/filters/common/ext_authz @esmet @gsagula @pradeepcrao @UNOWNED
+/*/extensions/filters/http/ext_authz @esmet @gsagula @pradeepcrao @UNOWNED
+/*/extensions/filters/network/ext_authz @esmet @gsagula @pradeepcrao @UNOWNED
+/*/extensions/filters/http/ip_tagging @UNOWNED @UNOWNED
+/*/extensions/filters/listener/original_dst @UNOWNED @UNOWNED
+/*/extensions/tracers/zipkin @UNOWNED @UNOWNED
+/*/extensions/tracers/dynamic_ot @UNOWNED @UNOWNED
+/*/extensions/tracers/opencensus @UNOWNED @UNOWNED
+/*/extensions/tracers/common @UNOWNED @UNOWNED
+/*/extensions/tracers/common/ot @UNOWNED @UNOWNED
+/*/extensions/filters/network/mongo_proxy @UNOWNED @UNOWNED
+
+# Intentionally exempt (treated as core code)
+/*/extensions/filters/common @UNOWNED @UNOWNED
+/*/extensions/filters/http/common @UNOWNED @UNOWNED
+/*/extensions/filters/network/common @UNOWNED @UNOWNED
+
+# To be moved to Contrib
```

Comment:
I can help with this if confirmed to be moved.

Reply:
Hi, I can also help with this.

Reply:
Awesome!  @daixiang0 want to take the first two and @giantcroc the last?  We may end up with others after giving the "unowned" ones a while to see if anyone picks them up

Reply:
Sounds great!